auto_submit/lib/requests/github_webhook.dart

```diff
@@ -17,7 +18,26 @@ class GithubWebhook {
   Future<Response> post(Request request) async {
     final Map<String, String> reqHeader = request.headers;
     logger.info('Header: $reqHeader');
+
+    //Listen to the pull request with 'autosubmit' label.
+    bool hasAutosubmit = false;
     final String rawBody = await request.readAsString();
+    final body = json.decode(rawBody) as Map<String, dynamic>;
+    List<Map<String, dynamic>> labels = <Map<String, dynamic>>[];
+    if (body.containsKey('pull_request') && body['pull_request'].containsKey('labels')) {
+      labels = List<Map<String, dynamic>>.from(body['pull_request']['labels']);
+    }
+
+    for (int i = 0; i < labels.length; i++) {
+      if (labels[i]['name'] == 'autosubmit') {
+        hasAutosubmit = true;
+      }
+    }
+
+    if (hasAutosubmit) {
+      //Creat a github client and use rest API to get this single pr
```

Comment:
nit: Make this a TODO

```
// TODO(kristinbi): Check if PR can be submitted. https://github.com/flutter/flutter/issue/###### 
```

1. https://dart-lang.github.io/linter/lints/flutter_style_todos.html